src/Authentication/JWTAuthenticationHandler.php

```diff
@@ -67,32 +56,28 @@ public function authenticateRequest(HTTPRequest $request)
      * Authenticate on every run, based on the header, not relying on sessions or cookies
      * JSON Web Tokens are stateless
      *
-     * @param Member $member
-     * @param bool $persistent
+     * @param Member           $member
```

Comment:
Matter of preference I guess? I'm not too fuzzed, but I prefer tight aligned for docblocks.

Not an issue for me though, just saying :)

Reply:
Normally I'm the one that fussed. Sorry I'll fix those. :)